src/frontend/src/stores/flowStore.ts

```diff
@@ -79,6 +79,9 @@ const useFlowStore = create<FlowStoreType>((set, get) => ({
   nodes: [],
   edges: [],
   isBuilding: false,
+  stopBuilding: () => {
+    get().buildController.abort();
```

Comment:
You should probably check if it exists first

Reply:
it always exists because it is initialized as a new AbortController